ADAL/src/request/ADAuthenticationRequest+Broker.m

```diff
@@ -306,14 +307,25 @@ - (void)callBroker:(ADAuthenticationCallback)completionBlock
     id<ADTokenCacheDataSource> dataSource = [_tokenCache dataSource];
     if (dataSource && [dataSource isKindOfClass:[ADKeychainTokenCache class]])
     {
+        NSString* keychainGroup = [(ADKeychainTokenCache*)dataSource sharedGroup];
+        NSString* teamId = [ADKeychainUtil keychainTeamId:&error];
```

Comment:
I traced through the code a little bit and I guess we're fine because you check the result length in `ADKeychainUtil.retrieveTeamIDFromKeychain`.

This leads me to another question: The query you use in `ADKeychainUtil.retrieveTeamIDFromKeychain` searches for a keychain item with `teamIDHint` but `ADKeychainTokenCache` inserts a keychain item with no label. Are these two separate systems?